django_mock_queries/query.py

```diff
@@ -30,8 +30,9 @@ def MockSet(*initial_items, **kwargs):
         'prefetch_related',
         'select_for_update'
     ])
-    mock_set.cls = clone.cls if clone else kwargs.get('cls', empty_func)
+    mock_set.cls = clone.cls if clone else kwargs.get('cls', MockModel)
```

Comment:
I did it, but had to do some tweaks with MockModel class.
'''python
    def __call__(self, *args, **kwargs):
        return MockModel(*args, **kwargs)
'''
Also create() now fills fields that are not specified in kwargs with None.